tests/data/retrieve/toh_metadata.json

```diff
@@ -0,0 +1,294 @@
+      "lastName": "Arnold",
+      "orcid": "0000-0001-6797-1715",
+      "self": {
+        "comments": [],
+        "uri": "http://meta.icos-cp.eu/resources/people/Sabrina_Arnold"
+      }
+    }, {
+      "firstName": "Matthias",
+      "lastName": "Lindauer",
+      "self": {
+        "comments": [],
+        "uri": "http://meta.icos-cp.eu/resources/people/Matthias_Lindauer"
+      }
+    }, {
+      "firstName": "Jennifer",
+      "lastName": "Müller-Williams",
+      "orcid": "0000-0002-4840-7729",
+      "self": {
+        "comments": [],
+        "uri": "http://meta.icos-cp.eu/resources/people/Jennifer_M%C3%BCller-Williams"
+      }
+    }, {
+      "firstName": "Marcus",
+      "lastName": "Schumacher",
+      "self": {
+        "comments": [],
+        "uri": "http://meta.icos-cp.eu/resources/people/Marcus_Schumacher"
+      }
+    }],
+    "citationBibTex": "@misc{https://hdl.handle.net/11676/y3-5_I70nW_F5PYN4i8m7WjO,\r\n  author={Kubistin, D., Plaß-Dülmer, C., Arnold, S., Lindauer, M., Müller-Williams, J., Schumacher, M.},\r\n  title={ICOS ATC CO2 Release, Torfhaus (147.0 m), 2017-12-12–2021-01-31},\r\n  year={2021},\r\n  url={https://hdl.handle.net/11676/y3-5_I70nW_F5PYN4i8m7WjO},\r\n  publisher={Atmosphere Thematic Centre},\r\n  copyright={http://meta.icos-cp.eu/ontologies/cpmeta/icosLicence},\r\n  pid={11676/y3-5_I70nW_F5PYN4i8m7WjO}\r\n}",
+    "citationRis": "TY - DATA\r\nT1 - ICOS ATC CO2 Release, Torfhaus (147.0 m), 2017-12-12–2021-01-31\r\nID - 11676/y3-5_I70nW_F5PYN4i8m7WjO\r\nPY - 2021\r\nUR - https://hdl.handle.net/11676/y3-5_I70nW_F5PYN4i8m7WjO\r\nPB - Atmosphere Thematic Centre\r\nAU - Kubistin, D.\r\nAU - Plaß-Dülmer, C.\r\nAU - Arnold, S.\r\nAU - Lindauer, M.\r\nAU - Müller-Williams, J.\r\nAU - Schumacher, M.\r\nER - ",
+    "citationString": "Kubistin, D., Plaß-Dülmer, C., Arnold, S., Lindauer, M., Müller-Williams, J., Schumacher, M., ICOS RI, 2021. ICOS ATC CO2 Release, Torfhaus (147.0 m), 2017-12-12–2021-01-31, https://hdl.handle.net/11676/y3-5_I70nW_F5PYN4i8m7WjO",
+    "licence": {
+      "baseLicence": "https://creativecommons.org/licenses/by/4.0",
+      "name": "ICOS CCBY4 Data Licence",
+      "url": "http://meta.icos-cp.eu/ontologies/cpmeta/icosLicence",
+      "webpage": "https://data.icos-cp.eu/licence"
+    },
+    "temporalCoverageDisplay": "2017-12-12–2021-01-31",
+    "title": "ICOS ATC CO2 Release, Torfhaus (147.0 m), 2017-12-12–2021-01-31"
+  },
+  "size": 580181,
+  "specificInfo": {
+    "acquisition": {
+      "instrument": [{
+        "comments": [],
+        "label": "CO2-CH4-H2O Picarro Analyzer",
+        "uri": "http://meta.icos-cp.eu/resources/instruments/ATC_457"
+      }, {
+        "comments": [],
+        "label": "CO2-CH4-H2O Picarro Analyzer",
+        "uri": "http://meta.icos-cp.eu/resources/instruments/ATC_271"
+      }],
+      "interval": {
+        "start": "2017-12-12T00:00:00Z",
+        "stop": "2021-01-31T23:00:00Z"
+      },
+      "samplingHeight": 147.0,
```

Comment:
I'm a little confused how "samplingHeight" is set to 147.0 but is expected to be "10m" in `test_icos_retrieve_no_local`?